pack/parallel/bad.json

```diff
@@ -0,0 +1,72 @@
+[
+		{
+				"alternate_of": "01004",
+				"back_flavor": "Agnes Baker may be just an unassuming waitress in this life, but in a previous life, in a time and age undreamed of in the modern world, she had been a powerful witch. It began when she found a strange artifact — a key of some kind — in a dusty collection of family belongings in her attic. When she touched it, the memories came flooding back, along with one word: \"Hyperborea.\"",
```

Comment:
Flavor should read: "Agnes Baker may be just an unassuming waitress in this life, but in a previous life, she had been a powerful witch. It began when she found a strange artifact - a key of some kind - in a dusty collection of family belongings in her attic. When she touched it, the memories came flooding back, along with one word: \"Hyperborea.\"" [One clause has been dropped, changing the em dashes to hyphens as per our style.]